plugins/Editing.jsx

```diff
@@ -469,10 +474,8 @@ class Editing extends React.Component {
             relationData.set('values', JSON.stringify(relationValues));
             Object.entries(relationUploads).forEach(([key, value]) => relationData.set(mapPrefix + key, value));
 
-            this.props.iface.writeRelations(this.editLayerId(this.state.selectedLayer), newFeature.id, relationData, (result, errorMsg) => {
-                if(result === false) {
-                    this.commitFinished(false, errorMsg);
-                } else if(result.success !== true) {
+            this.props.iface.writeRelations(this.editLayerId(this.state.selectedLayer), newFeature.id, relationData, (result) => {
+                if(result.success !== true) {
```

Comment:
This looks like an incorrect rebase, can you please fix it?

Reply:
sorry